app/lib/brexit_checker/actions.yaml

```diff
@@ -1479,25 +1485,27 @@ actions:
   audience: business
 - id: T115
   priority: 2
-  title: Register with HSE to find out when guidance on the duties for classifying,
-    labelling and packaging chemicals for the GB market is available
-  consequence: You must make sure chemicals are classified, labelled and packaged
-    according to GB regulations.
+  title: Make sure chemicals are classified, labelled and packaged according to GB
+    regulations
+  consequence: You will not be able to place chemicals on the market unless they follow
+    the classification, labelling and packaging regulations.
   guidance_prompt: More information
-  guidance_link_text: 'Brexit: Chemicals industry guidance'
-  guidance_url: https://www.hse.gov.uk/brexit/chemicals-brexit-guidance.htm?utm_source=gov.uk&utm_medium=referral&utm_campaign=eu-transition&utm_content=checker-tool
+  guidance_link_text: Chemicals classification, labelling and packaging (CLP) after
+    the transition period
+  guidance_url: https://www.hse.gov.uk/brexit/clp.htm?utm_source=gov.uk&utm_medium=referral&utm_campaign=eu-transition&utm_term=clp&utm_content=checker-tool
   criteria:
   - chemical
   audience: business
 - id: T116
   priority: 2
-  title: Register with HSE to find out when guidance on processes for biocidal active
-    substance approval and product authorisation is available
-  consequence: You may have delays in getting access to the right materials to continue
-    with your work.
+  title: Get approval and product authorisations for new biocidal active substances
+    for the GB market from 1 January 2021
+  consequence: You will not be able to place new biocidal active substances on the
+    GB market without approval.
   guidance_prompt: More information
-  guidance_link_text: 'Brexit: Chemicals industry guidance'
-  guidance_url: https://www.hse.gov.uk/brexit/chemicals-brexit-guidance.htm?utm_source=gov.uk&utm_medium=referral&utm_campaign=eu-transition&utm_content=checker-tool
+  guidance_link_text: 'Biocides: What you''ll need to do after the transition period
+    ends'
+  guidance_url: https://www.hse.gov.uk/brexit/biocides.htm?utm_source=gov.uk&utm_medium=referral&utm_campaign=eu-transition&utm_term=biocides&utm_content=checker-tool'
```

Comment:
There's a single quote mark at the end of this line.  Not sure if it should be there?

Reply:
I don't know if it's related to the `you''ll` on line 1506?

Reply:
Good spot, ta!

Reply:
I think the one on the line above is yaml escaping. It looks OK in the [review app](https://finder-front-updates-la-d4n0cv.herokuapp.com/transition-check/results?c%5B%5D=chemical&c%5B%5D=import-from-eu&c%5B%5D=export-to-eu&c%5B%5D=owns-operates-business-organisation-uk&c%5B%5D=owns-operates-business-organisation&c%5B%5D=travel-eu-business-no&c%5B%5D=working-uk&c%5B%5D=living-uk&c%5B%5D=nationality-uk). 

The link one looks like a copy/paste issue and has been fixed in the source spreadsheet now

Reply:
I've pushed the fix